src/modules/addComment.js

```diff
@@ -0,0 +1,16 @@
+const displayComment = (newComment) => {
+  const today = new Date().toISOString().slice(0, 10);
+  const commentContent = document.querySelector('.result');
+  commentContent.innerHTML += `<div>${today} ${newComment.username} : ${newComment.comment}</div>`;
+};
+
+const addComment = async (newComment, url) => {
+  await fetch(url, {
+    method: 'POST',
+    body: JSON.stringify(newComment),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  }).then(displayComment(newComment));
+};
```

Comment:
:ballot_box_with_check: Please i noticed that on your async functions you don't take into account the error cases. please kindly use try{} and catch{} block  to handle error if  happens.